packages/core/src/modules/credentials/formats/CredentialFormatServiceOptions.ts

```diff
@@ -89,6 +93,14 @@ export interface FormatAcceptRequestOptions<CF extends CredentialFormat> {
   offerAttachment?: Attachment
 }
 
+export interface FormatAcceptCredentialOptions<CF extends CredentialFormat> {
+  credentialRecord: CredentialExchangeRecord
+  credentialFormats?: CredentialFormatPayload<[CF], 'acceptRequest'>
```

Comment:
`acceptCredential` doesn't need any input right? If we accept the credential it means we'll send an acknowledgement.